core/Piranha.Manager.Localization/Piranha.Manager.Localization.csproj

```diff
@@ -8,6 +8,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Localization" Version="2.2.0" />
+    <PackageReference Include="Microsoft.NET.Sdk.Razor" Version="3.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.3.0" />
```

Comment:
why you did these changes in all Piranha csproj files? if you havent done them - revert please